lib/Services/Request.ts

```diff
@@ -0,0 +1,14 @@
+import { Context, Data, Effect } from "effect";
+
+export class RequestJsonError extends Data.TaggedError("RequestJsonError")<{
+  error: unknown;
+}> {}
+
+export const Request = Context.Tag<Request>("Request");
```

Comment:
Hi @tim-smart 

With the latest Effect I'm getting an error with this:
`Type 'Request' does not satisfy the constraint 'string'`.

What should this be now?

Thanks